lib/src/resultset-table/ResultsetTable.tsx

```diff
@@ -119,7 +119,7 @@ const DxcResultsetTable = ({
           </thead>
           <tbody>
             {filteredResultset.map((cells, index) => (
-              <tr key={`resultSetTableCell_${index}`}>
+              <tr key={`resultSetTableCell_${page * itemsPerPage + index ?? index}`}>
```

Comment:
I think this can be simplified to resultSetTableCell_page_index.

Reply:
Yup, looks much neater.